tools/test-js-with-node

```diff
@@ -154,6 +154,7 @@ EXEMPT_FILES = make_set(
         "web/src/navbar_help_menu.js",
         "web/src/navbar_menus.js",
         "web/src/navigate.js",
+        "web/src/onboarding_steps.js",
```

Comment:
If the approach looks good, I'd quickly add coverage for this. Just a temporary addition.

Reply:
Also, I'd write events docs in a better way, after getting a review pass on it; if the approach is the same as expected.